test/general/header_inclusion_order_memory_1.pass.cpp

```diff
@@ -20,9 +20,22 @@
 
 #include "support/utils.h"
 
+#if ONEDPL_HAS_RANGE_ALGORITHMS > 202505L
```

Comment:
How does this (bad) test differ from the previous one, which is `_memory_0.pass`?

Reply:
No difference. Just checking that the same thing works if we change the order of the inclusion of the headers.